microsoft-edge/extensions-chromium/developer-guide/alternate-distribution-options.md

```diff
@@ -160,6 +166,24 @@ The policies for `ExtensionInstallForcelist`, block list, and `ExtensionSettings
 If the `BlockExternalExtensions` policy is set to `Not Configured` or `Disabled`, that doesn't have any effect.  If the `BlockExternalExtensions` policy is set to `Enabled`, external extension installation (by third party software) is prevented.
 
 
+<!-- ====================================================================== -->
+## See also
+<!-- all links in article -->
+
+* [Set an externally installed extension to automatically update](../publish/auto-update.md)
+* [Manifest file format for extensions](../getting-started/manifest-format.md)
+
+Microsoft Edge Enterprise docs:
+* [Microsoft Edge - Policies](/deployedge/microsoft-edge-policies)
+   * [ExtensionInstallForcelist](/deployedge/microsoft-edge-policies#extensioninstallforcelist)
+   * [ExtensionInstallSources](/deployedge/microsoft-edge-policies#extensioninstallsources)
+   * [ExtensionSettings](/deployedge/microsoft-edge-policies#extensionsettings)
+   * [BlockExternalExtensions](/deployedge/microsoft-edge-policies#blockexternalextensions)
+
+Chrome Extensions docs:
```

Comment:
exposed the Chrome docs "[here]" link in the body of the article (h2 See Also at bottom).  usually don't mention Chrome docs in article body in such case, but this could help for maintenance (inspecting the two versions of article)
added "todo" note, to consider this design

Reply:
pretty innocuous to add such link, it will help Maint.  closing cmt